### PR TITLE
feat: Add migrations and disclosures auto trigger

### PR DESCRIPTION
**Pull request by calldelegation**
# Summary

Adding an auto trigger to update release notes immediately after release
This line specifically:
https://github.com/FuelLabs/migrations-and-disclosures/blob/d479dcd74fffabc9156b9e0ede8aea8343d99454/.github/workflows/migrations.yml#L9-#L10

# Checklist

- [ ] I **_added_** — `tests` to prove my changes
- [ ] I **_updated_** — all the necessary `docs`
- [x] I **_reviewed_** — the entire PR myself, using the GitHub UI
- [ ] I **_described_** — all breaking changes and the Migration Guide

### CODE REVIEW

**Review by danielbate — COMMENTED** [review]
Could this not go in `.github/workflows/release.yaml`? Seems overkill to have it's own workflow.

**Comment by petertonysmith94** [comment]
> Could this not go in `.github/workflows/release.yaml`? Seems overkill to have it's own workflow.

[1a77f44](https://github.com/FuelLabs/fuels-ts/pull/2844/commits/1a77f44527973981d30cb5e28c0a09a5e2bfaa8c)

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -202,3 +202,11 @@ jobs:
           aws s3 cp ./packages/account/src/providers/assets/images/ s3://${S3_CDN_BUCKET}/assets/ --recursive
         env:
           S3_CDN_BUCKET: ${{ secrets.S3_CDN_BUCKET }}
+
+      - name: Notify migrations and disclosures of the new release (breaking changes)
+        run: |
+          curl -X POST \
+            -H "Accept: application/vnd.github.v3+json" \
+            -H "Authorization: token ${{ secrets.TARGET_REPO_TOKEN }}" \
```
[the remark this answers is not in the document]
Comment (arboleya) [reply .github/workflows/release.yaml:210]: Not yet.

[the remark this answers is not in the document]
Comment (arboleya) [reply .github/workflows/release.yaml]: Token added and updated via https://github.com/FuelLabs/fuels-ts/pull/2844/commits/6c593b6cd93dabdd28b9768ad0f53dde2de403b0.

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -202,3 +202,11 @@ jobs:
           aws s3 cp ./packages/account/src/providers/assets/images/ s3://${S3_CDN_BUCKET}/assets/ --recursive
         env:
           S3_CDN_BUCKET: ${{ secrets.S3_CDN_BUCKET }}
+
+      - name: Notify migrations and disclosures of the new release (breaking changes)
```
Comment (arboleya) [inline .github/workflows/release.yaml:206]: Will this pull the new docs as well?

**Review by arboleya — APPROVED** [review]
It only gets better. 🎉

**Review by petertonysmith94 — APPROVED** [review]
Thank you @calldelegation 🌟

**Review by danielbate — APPROVED** [review]

File: .github/workflows/release.yaml (hunk 2 above)
Comment (petertonysmith94) [reply .github/workflows/release.yaml:206]: Looks like it'll only update the `migrations-and-disclosures` [repository](https://github.com/FuelLabs/migrations-and-disclosures/blob/f6123525f6c58f01e4717bda1ecd7ca9d70eab45/.github/workflows/migrations.yml#L9-L10) with our version.

I assume then the process of updating the `docs-hub` is then manual from there.

**Pull request merged**